testimony/__init__.py

```diff
@@ -444,6 +448,93 @@ def validate_docstring_report(testcases):
         return -1
 
 
+def validate_values_report(testcases):
+    """Check that given token contains one of allowed values."""
+    result = {}
+    invalid_docstring_count = 0
+    invalid_token_value_count = 0
+    missing_docstring_count = 0
+    missing_token_count = 0
+    testcase_count = 0
+    for path, tests in testcases.items():
+        testcase_count += len(tests)
+        for testcase in tests:
+            issues = []
+            missing_token_count_local = 0
+            if not testcase.docstring:
+                issues.append('Missing docstring.')
+                missing_docstring_count += 1
+            for token in SETTINGS['tokens']:
+                token = token.lower()
+                token_config = SETTINGS['token_configs'][token]
+                if token not in testcase.tokens:
+                    missing_token_count_local += 1
+                    missing_token_count += 1
+                if token in testcase.tokens \
+                   and not token_config.validate(testcase.tokens[token]):
+                    issues.append(
+                        'Token {} have unexpected value of {}'.format(
+                            token, testcase.tokens[token])
+                    )
+                    invalid_token_value_count += 1
+            if missing_token_count_local > 0:
```

Comment:
What about instead of counting creating a list of missing tokens so the message could provide which tokens are missing?

Reply:
I believe this is done in #142 :

```
ConfigurationFileTestCase::test_multiple_invalid_keys:202
---------------------------------------------------------

* Tokens with invalid values:
  Caseimportance: Lowest (type: 'choice')
  Status: Invalid (type: 'choice')
```

If I misunderstood, please let me know in other PR.